python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -553,6 +553,42 @@ def test_drop_na(self):
             self.spark.sql(query).na.drop(how="any", thresh=2, subset="a").toPandas(),
         )
 
+    def test_replace(self):
```

Comment:
* can you add some negative tests for assertions?

Reply:
will add